cmake/FindDiscordRpc.cmake

```diff
@@ -0,0 +1,27 @@
+# discord-rpc is a rare case where vcpkg doesn't provide a proper mean to find_package() out-of-the-box.
+# As the project itself is discontinued there's no hope for PR #345 to be merged
+# so we're stuck at manually finding libraries.
```

Comment:
We could also upgrade to the newer discord library that isn't discontinued but this will require other changes in Hyperspace too, might be suited for a future PR although I myself don't care about Discord but maybe after this PR one could be stood up to change to the new Discord library in the build and the other HS devs could try to switch to it?

Reply:
True, it doesn't worth assigning priority on this for now.

Reply:
I created #22 for now